packages/react/package.json

```diff
@@ -339,7 +339,7 @@
     "whatwg-fetch": "^3.0.0"
   },
   "sideEffects": false,
-  "version": "2.148.0-next.15",
+  "version": "2.148.0",
```

Comment:
I'm not sure about these versions. Master is `2.148.0`, but if you're merging `master` back into `next`, I think `next` keeps its version, because it's still _newer_ than `master`. Don't all of the changes that exist in `next` that haven't made it into `master` yet cause it to still have a new version number? 🤔

Reply:
... Or, is this correct, because now you're bringing a "completed" `2.148.0` merge into `next`, so the next published version of `next` will become `1.248.1-next.0`--which seems correct.... Again, I'm not sure. Just double checking. 🤔

Reply:
lol. I had the same question above. I think Bjorn may be right and when we merge out automation will bump the version up like you describe here.

Reply:
I think your second comment is how it should work, discussed this with David yesterday as well. We are all a bit confused here hehe. But looking at the history of the CHANGELOG files the 2.x.y always lands above the 2.x.y-next.1,  2.x.y-next.2 etc

Reply:
See this old commit for 2.147.0
https://github.com/carbon-design-system/carbon-addons-iot-react/commit/0bcc680eebf4398f001aa99df0625e5f28848dde